packages/tabs/src/Tabs.tsx

```diff
@@ -149,7 +150,9 @@ function Tabs({
   };
 
   const handleArrowKeyPress = (e: KeyboardEvent) => {
-    if (!(e.metaKey || e.ctrlKey)) {
+    const isFocusedTabs = tabsRef?.current?.contains(document.activeElement);
```

Comment:
isn't the problem that we're using `useEventListener` to bind the event handler to the document instead of binding it just to this component? if the handler is bound to the component this condition would always be true.

Reply:
👍